modules/ort/src/main/kotlin/workflow/processors/enricher/OrtDownloaderProcessor.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.ort.workflow.processors.enricher
+
+import com.here.ort.downloader.DownloadException
+import com.here.ort.downloader.Downloader
+import com.here.ort.utils.encodeOrUnknown
+import com.here.ort.utils.packZip
+import org.eclipse.sw360.antenna.model.artifact.Artifact
+import org.eclipse.sw360.antenna.model.artifact.facts.ArtifactSourceFile
+import org.slf4j.LoggerFactory
+import org.eclipse.sw360.antenna.ort.utils.ArtifactToPackageMapper
+import java.io.File
+import org.eclipse.sw360.antenna.api.workflow.AbstractProcessor
+import org.eclipse.sw360.antenna.model.artifact.ArtifactCoordinates
+import org.eclipse.sw360.antenna.model.artifact.facts.ArtifactFilename
+
+class OrtDownloaderProcessor : AbstractProcessor() {
+    companion object {
+        const val ORT_DOWNLOADER_DIR = "ort-downloader-result"
+        private val LOGGER = LoggerFactory.getLogger(OrtDownloaderProcessor::class.java)
+    }
+
+    init {
+        workflowStepOrder = 350
+    }
+
+    private lateinit var ortDownloadFolder: File
```

Comment:
I believe the variable name is misleading as this is not the directory ORT downloads to, but the directory you ZIP the sources to after ORT has downloaded them. Maybe better: `sourcesZipFolder`. As a side note, I personally prefer the term "directory" over "folder" as the latter is mostly used in the Windows world whereas the former is more neutral.